src/app.js

```diff
@@ -1 +1,45 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const [, , sourceFile, destinationFile] = process.argv;
```

Comment:
```suggestion
const [_, __ , sourceFile, destinationFile] = process.argv;
```
But best way use slice for this case